torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -176,7 +172,14 @@ TypePtr SchemaTypeParser::parseRefinedTensor() {
       dims.push_back(dim);
     });
     at::IntArrayRef dims_ref(dims);
-    ptr = at::TensorType::create(dtype, at::DeviceType::CPU, dims_ref, false);
+    ptr = at::TensorType::create(
+        dtype,
+        at::DeviceType::CPU,
+        c10::VaryingShape<c10::ShapeSymbol>(dims_ref),
```

Comment:
Implementation details of TensorType appear to have leaked here. I think it would look clearer of stride information was represented as its own object.